### PR TITLE
Use newest Firefox for integration tests with geckodriver

### PR DESCRIPTION
**Pull request by oliverguenther**

### CODE REVIEW

File: spec/support/capybara.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
   # need to disable marionette as noted
   # https://github.com/teamcapybara/capybara#capybara
```
Comment (ulferts) [inline spec/support/capybara.rb]: Please remove the comment

File: .travis.yml [hunk 2]
```diff
@@ -107,7 +113,7 @@ notifications:
       - secure: "dkVCNYSedLfeuqTboRPx/iWmXldhoHZpSp83cpkG6ZL81Psut2ZEHg6aBiWgqQWdfNAca4a51hWXs6utRqdgnxo51YAB/pMy0/b+jEOjZvuShOVz6TVqn5W5JLTNZJeVZL+BgvwXZWbIujH/cSuUv+h7GTnS+8Y4b2ZrCYilP35sAck2bbbAsKzbuW2yZZqK7X0UnSjoxg+KeT9DuK1zjjJJ6CZig1/n5ZjciQIv7JD50hmqFxBgjyBi71Loh/iohUF89EoQRXm0KkSmEhCINld5s+7SP4jSmnqyrNamNNw2ZGCgg3OXhcvDqOHjPwpCNNhrtzTExt/Z2cTzHz4FnV9ncJj9i1pkbjveuudVUTeaSHHXjw/X496OKLP3LwEZYggUCyc3nu9qR2zSs6zdG4ijhclOVzNrXDDixjpYGmOg8mVunG2cwFUSjtmX8nyPNXQvRnvsfnVYalprqXJHGmQW07nFE/2lbjdmnQiCEExNsKXO5vqrqo6RYOtTfka9B5s3ZkTkXgZkTWuEgWbaMC/fZhi0NCu2Ow3XOzQrs8hVLF4w1zngjqOLjk+LftqWwT42D47RKW3SZfVEVm8Vw+JGfh/uJUNAsPsu+Mybw8k0aDmLjPTlRZD+dYvQ2EPizgc2+cf+0GVoaGzKZF+cJ8I0ut8kO13M9ZmjiHU0se4="
 
 addons:
-  firefox: "45.0esr"
+  firefox: "latest"
```
Comment (ulferts) [inline .travis.yml]: why not the latest `esr` as this is what we officially support?

Comment (oliverguenther) [reply .travis.yml]: Latest ESR is 52, too old for testing with Marionette (things like double_click , send_keys will not work properly).

**Comment by ulferts** [comment]
Works fine. I fixed a typo.

I also piggybacked the PR to remove cukes that have been deactivated three years ago.

**Pull request merged**